tpl/default/register.tpl

```diff
@@ -115,31 +115,38 @@
+{t}The STK Addons service is designed to be a repository exclusively for SuperTuxKart addon content. All uploaded content must be intended for this purpose. When you upload your content, it will be available publicly on the internet, and will be made available in-game for download.{/t}
 
-{t}Super Tux Kart aims to comply with the Debian Free Software Guidelines (DFSG). TuxFamily.org also requires that content they host comply with open licenses. You may not upload content which is locked down with a restrictive license. Licenses such as CC-BY-SA 3.0, or other DFSG-compliant licenses are required. All content taken from third-party sources must be attributed properly, and must also be available under an open license. Licenses and attribution should be included in a "license.txt" file in each uploaded archive. Uploads without proper licenses or attribution may be deleted without warning.{/t}
+{t}SuperTuxKart aims to comply with the Debian Free Software Guidelines (DFSG). TuxFamily.org also requires that content they host comply with open licenses. You may not upload content which is locked down with a restrictive license. Licenses such as CC-BY-SA 3.0, or other DFSG-compliant licenses are required. All content taken from third-party sources must be attributed properly, and must also be available under an open license. Licenses and attribution should be included in a "license.txt" file in each uploaded archive. Uploads without proper licenses or attribution may be deleted without warning.{/t}
 
-{t}Even with valid licenses and attribution, content may not contain any of the following:{/t}
-1. {t}Profanity{/t}
-2. {t}Explicit images{/t}
-3. {t}Hateful messages and/or images{/t}
-4. {t}Any other content that may be unsuitable for children{/t}
-{t}If any of your uploads are found to contain any of the above, your upload will be removed, your account may be removed, and any other content you uploaded may be removed.{/t}
+{t}Even with valid licenses and attribution, neither addon content, nor usernames, nor any other content, shall contain any of the following:{/t}
 
-{t}By checking the box below, you are confirming that you understand these terms. If you have any questions or comments regarding these terms, one of the members of the development team would gladly assist you.{/t}
+{t} - Profanity{/t}
+
+{t}- Explicit content{/t}
+
+{t} - Hateful messages and/or content{/t}
+
+{t} - Any other content that may be considered unsuitable for children{/t}
+
+{t}If any of your content is found to violate these terms, any of your uploads, and/or your account, may be removed at any time.{/t}
+
+{t}Moderators hold the right to change, add, or remove any content at any time, with or without notice. These conditions may change at any time, with or without notice. In the event of a change of these terms, continuing to use your account indicates your acceptance of the new/revised terms.
```

Comment:
"In the event of a change of these terms, continuing to use your account indicates your acceptance of the new/revised terms."

I'm not sure that is legal

Reply:
From Google's ToS (https://policies.google.com/terms?hl=en&gl=ZZ):

> We may modify these terms or any additional terms that apply to a Service to, for example, reflect changes to the law or changes to our Services. You should look at the terms regularly. We’ll post notice of modifications to these terms on this page. We’ll post notice of modified additional terms in the applicable Service. Changes will not apply retroactively and will become effective no sooner than fourteen days after they are posted. However, changes addressing new functions for a Service or changes made for legal reasons will be effective immediately. If you do not agree to the modified terms for a Service, you should discontinue your use of that Service.

Maybe my text could be tweaked but the concept seems the same.

Reply:
I am not sure about this.
@hiker opinions?

Reply:
The questionable section has been removed, and some more fixes were made so that TOS properly reflects the actual policies currently used by admins (cheating the ranking system is prohibited, non-DFSG is accepted as long as license allows redistribution and modification though CC-BY-SA still recommended, moderators can update/remove content, no longer hosted on TuxFamily, etc.)